src/_listObject.scss

```diff
@@ -0,0 +1,228 @@
+    }
+
+    &--top-left {
+      flex: 3 3 10%;
+      width: 60%;
+
+      @include fd-rtl() {
+        text-align: right;
+      }
+    }
+
+    &--top-right {
+      width: 40%;
+      flex: 2 2 10%;
+      text-align: right;
+      padding-left: $fd-list-object-with-padding-value-one !important;
+      vertical-align: $fd-list-object-with-vertical-align-top !important;
+      margin-right: $fd-list-object-with-margin-value-zero !important;
+
+      @include fd-rtl() {
+        text-align: left;
+        padding-right: $fd-list-object-with-padding-value-one !important;
+        padding-left: $fd-list-object-with-padding-value-zero !important;
+        margin-left: $fd-list-object-with-margin-value-zero !important;
+      }
+    }
+
+    &--bottom-row {
+      display: flex;
+      align-items: flex-end;
+      justify-content: flex-end;
+
+      > div:only-child {
+        padding: 0;
+      }
+    }
+
+    &--bottom-left {
+      flex: 3 3 10%;
+      text-align: left;
+      padding: 0.5rem 0.5rem 0.5rem 0;
+      display: inline-grid;
+
+      @include fd-rtl() {
+        text-align: right;
+        padding: 0.5rem 0 0.5rem 0.5rem;
+      }
+    }
+
+    &--bottom-right {
+      width: 50%;
+      max-width: 50%;
+      flex: 2 2 10%;
+      text-align: right;
+      margin-right: $fd-list-object-with-margin-value-zero;
+      padding: 0.5rem 0 0.5rem 0.5rem;
+      display: inline-grid;
+
+      @include fd-rtl() {
+        text-align: left;
+        margin-left: $fd-list-object-with-margin-value-zero;
+        padding: 0.5rem 0.5rem 0.5rem 0;
+      }
+    }
+
+    &--bottom {
+      margin-top: 0.25rem;
+    }
+  }
+
+  .fd-object-number {
+    vertical-align: $fd-list-object-with-vertical-align-top !important;
+    margin-right: $fd-list-object-with-margin-value-zero;
+
+    &__text {
+      font-family: $fd-list-object-number-font-family;
+      font-weight: $fd-list-object-font-weight;
+      font-size: 1.375rem;
+    }
+
+    &__unit {
+      font-family: $fd-list-object-number-font-family;
+      font-size: $fd-list-object-font-size;
+    }
+
+    &__text,
+    &__unit {
+      padding-top: $fd-list-object-with-padding-value-zero !important;
+      display: block !important;
+    }
+
+    &__text::after {
+      content: none;
+    }
+
+    @include fd-rtl() {
+      margin-left: $fd-list-object-with-margin-value-zero;
+    }
+  }
+
+  .fd-object-identifier {
+    &__title {
+      @include fd-reset();
+
+      font-family: var(--sapFontHeaderFamily);
+      font-size: var(--sapFontHeader4Size);
+      font-weight: var(--sapFontHeaderWeight);
+      color: var(--sapGroup_TitleTextColor);
+      line-height: 1.5;
+      vertical-align: top;
+      display: -webkit-box;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
+      overflow: hidden;
+      white-space: normal;
+      text-overflow: ellipsis;
+
+      @include fd-rtl() {
+        float: right;
+      }
+    }
+  }
+
+  .fd-object-status {
```

Comment:
Is it allowed to use external selectors for styling?
Should not we create our ourn selector for `fd-object-status`?

Reply:
we need all the styles of fd-object-status, only a few values in it are changing in the object list, so instead of rewriting whole styles again. I reusing the class and overriding the required values. The same is done where ever it is applicable.

Reply:
@InnaAtanasova could you please tell if such approach is acceptable here?

Reply:
updated it as per Inna's commets now